core/views.py

```diff
@@ -1,9 +1,9 @@
-from django.http import HttpResponse
```

Comment:
Why you deleted that line, when you commented line 8 instead of also deleting that?